steamcmd_servers/space_engineers/README.md

```diff
@@ -0,0 +1,27 @@
+# Space Engineers
+Space Engineers is a voxel-based sandbox game set in space and on planets. 
+
+Many parts of this egg are based on the work of [mmmaxwwwell](https://github.com/mmmaxwwwell/space-engineers-dedicated-docker-linux). A thank you therefore goes to him.
+
+**NOTE**
+Because of the winetricks installation, the install process could take some time. Likewise, the first start of the server takes its time.
+
+When installed, the Server loads a base World and config from this repo. New world must be generated with the Windows Dedicated Server and then uploaded to the /config/Saves folder.
+Everything which is not defined in a variable could be edited directy in the config file /config/SpaceEngineers-Dedicated.cfg.
+
+## Minumum server settings
+### RAM
+This server requires about 6GB.
+
+Based on the information provided by the developer: [See here](https://www.spaceengineersgame.com/dedicated-servers/)
+
+### Disk
+This server uses about 7GB of diskspace.
+
+## Server Ports
+
+| Port  | default |
+|-------|---------|
+| Game  | 27016   |
+| Steam | 8766    |
+| API   | 8080    |
```

Comment:
The default Deamon Port is 8080 and if someone assign it, will break the deamon

Reply:
Oops. Will change it. Thanks.